internal/blockchain/error.go

```diff
@@ -583,6 +583,14 @@ const (
 	// network parameter deployment definitions.
 	ErrDuplicateDeployment = ErrorKind("ErrDuplicateDeployment")
 
+	// ErrUnknownDeploymentChoice indicates a choice id for a deployment does
```

Comment:
```suggestion
	// ErrUnknownDeploymentChoice indicates the forced choice id for a deployment does
```

Reply:
Maybe the new error names should also be updated to include `Forced`, although that would make them pretty verbose.

Reply:
Good observation.  However, I actually intentionally left it as the more generic error, because as discussed elsewhere, I would like to ultimately get rid of the whole `invalidThresholdState`/`invalidChoice` and change the semantics of choices to use their ID instead of an index into the config params as it is today.  That will further simplify the logic, but it will mean this error applies to more cases than just the forced one.